### DISCUSSION TITLE
Allow merging into synonyms

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5994] [Feature Requests]
**Is your feature request related to a problem? Please describe.**  
RBGE says it would be helpful because they often face challenges when merging taxa, particularly when one of the taxa is a synonym. Currently, if they need to merge a non-synonymised taxon into a synonym, they must first de-synonymise the name, perform the merge, and then re-synonymise it. This multi-step process is cumbersome and increases the risk of errors, _especially_ during the re-synonymisation step.

**Describe the solution you'd like**  
RBGE proposes a solution that includes a preference option allowing users to merge non-synonymised names directly into synonyms without the need for de-synonymisation. This would streamline their workflow and reduce the potential for mistakes.

**Describe alternatives you've considered**  
While RBGE has considered the current method of de-synonymising, merging, and re-synonymising, they find it inefficient and prone to errors. 

They have also thought about the possibility of merging two synonyms, which could lead to conflicts with accepted names. In such cases, RBGE believes the newly merged name should retain the synonym of the name it was merged into. This may require manual resolution to ensure that correct synonymy is maintained.

**Reported By**  
Robyn at Royal Botanic Garden Edinburgh (RBGE) on the [Speciforum](https://discourse.specifysoftware.org/t/merging-into-synonyms-in-taxon-tree/2090)

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/merging-into-synonyms-in-taxon-tree/2090/2
